style.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: darkgreen; 
+}
```

Comment:
- [ ] Great job here 💯, but according to the requirements you should use a `class` to select the `<h1>` tag, so kindly add a class to your `<h1 class="header">` tag, then use this class in your CSS file 👍
```
.header {
 color: darkgreen;
}
```
![1](https://user-images.githubusercontent.com/45231276/198199259-0ed4bf8a-cc08-48fa-ad50-67a31bea2891.PNG)